content/06.discussion-a.md

```diff
@@ -1,82 +1,69 @@
-When use of GitHub is taught within an EEB context, it is usually accompanying coursework in topics such as statistical programming, and some students may find it overwhelming to juggle learning git alongside scripting languages, statistical theory, and file system navigation, especially when many may also be new to using command-line interfaces in general. 
-Instructors likewise may confuse the expected digital literacy of younger students with computational fluency, even when modern technology increasingly abstracts many relevant concepts through search optimization and preponderant IDEs.
-The default public nature of GitHub usage can add additional pressure to students learning to use the platform.  
-While many quantitative ecologists and evolutionary biologists take advantage of GitHub for individual use, collaborative use may lag due to how researchers traditionally divide labor within projects. 
-Despite broad utility, GitHub remains a tool predominantly used by computer scientists and software developers, and EEB researchers may take the view that GitHub is a platform that only needs to be used by individuals writing code, and may silo those aspects of projects to a single individual.    
-Those assumptions may have obscured the utility of GitHub for tasks other than traditional data analysis and software development, or how GitHub can facilitate the integration of code with non-coding aspects of projects through the practice of repository design. 
-Additional hesitancy may come from general reluctance to share data and code publicly, or technical and logistical issues related to storage of large data files and lack of integration with other research platforms. 
-
-### Our own limitations in writing this paper
-
-*Contributors to this section: Ali*
+There are also more logistical constraints to using GitHub in research workflows.  
+Due to some file size limitations imposed by GitHub it can be difficult to synch large files to GitHub repos.  
+One way around this issue is to attach additional large files to the repository release once code in the repository is ready for use.  
+Another limitation is the lack of GitHub help documents for non-English researchers in EEB that may lead them to miss the opportunity to fully understand the importance version control as well as the other benefits of GitHub.  
+Lastly, GitHub is owned by a private company (microsoft) and though the platform hosts an immense amount of open source code, researchers should also consider archiving research materials in long-term data and code repositories.  
 
-Our own limitations since we are mostly writing from the EcoEvo perspective/ additional github limitation
-Reliance on R since we are generally in EcoEvo
-Discussion of free vs. paid plans. When projects get highly collaborative may have to add / pay for accounts. At this point, little difference between paid and free.
+### Why aren't more EEB researchers using GitHub?
```

Comment:
The edits here are great, but it feels like it belongs better in the introduction.  Fine for now, but let's revisit in our big read-through.

Reply:
Sounds good, I think that's the type of big picture changes we can make in our next readthrough.